### ISSUE TITLE
Please default ESpeak NG's voice role to "Chinese (Mandarin, latin as Pinyin)" for Chinese to fix #1028

### ISSUE THREAD

**Issue by cary-rowen** [espeak-ng/espeak-ng#1163]
### Is your feature request related to a problem? Please describe.

#1028  makes ESpeak completely unusable for Chinese users. This is a disaster for NVDA beginners who cannot use Windows OneCore.
### Describe the solution you'd like
After testing, setting the voice role of ESpeak NG to "Chinese (Mandarin, latin as Pinyin)" can temporarily solve #1028

The word "temporarily" is used because the English pronunciation of "Chinese (Mandarin, latin as Pinyin" is less vivid than the current default "Chinese (Mandarin, latin as English)".
But this solution can get rid of the dilemma of no being able to use ESpeak NG at all.

### Describe alternatives you've considered
The best solution is to solve #1028  by ESpeak.
Make "Chinese (Mandarin, latin as English)" can speak Chinese correctly.
### Additional context.
I think this question is urgent enough, especially for beginners of nvda.

I created an issue in NVDA Repo [#13572](https://github.com/nvaccess/nvda/issues/13572)
But I was told that the problem should be solved by ESpeak.

**Comment by cary-rowen** [comment]
Does anyone pay attention to this issue? Chinese users are currently unable to use ESpeak on NVDA.

**Comment by cary-rowen** [comment]
Hello,
@hozosch 
@valdisvi 

Can you provide some useful information to advance this issue?

@seanbudd 

ESpeak is the default speech synthesizer for NVDA, I would also like to know if you have any solution for this, NVDA users who use Chinese are troubled by this, and often ask questions related to this in the community.

Thanks
